adapters/vuejs/src/single-spa-vue.js

```
@@ -57,6 +50,10 @@ function bootstrap(opts, mountedInstances, props) {
         fragmentName = props.name;
     }
 
+    if (props.errorHandler) {
```

I think if props.errorHandler is undefined you need to throw an error

---

And in any case this code will fail if here you will take else path ;)